spec/api/query_spec.rb

```diff
@@ -1,60 +1,42 @@
 require 'spec_helper'
```

Comment:
A lot of the previous decisions remain unclear to me. :/ Yes, this should be combined with search_spec. I would rather perform that merger after #165 is merged. #165 includes updates to search_spec, and I'd rather focus my efforts on finishing #165 than trying to merge these files.